### PR TITLE
Add llama2 configs for GPU A3

### PR DESCRIPTION
**Pull request by michelle-yooh**

### CODE REVIEW

File: MaxText/configs/a3/llama_2_7b/1vm.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ --xla_gpu_reduce_scatter_combine_threshold_bytes=67108864 --xla_gpu_enable_pipelined_all_gather=true \
+ --xla_gpu_enable_pipelined_reduce_scatter=true --xla_gpu_enable_pipelined_all_reduce=true \
+ --xla_gpu_enable_while_loop_double_buffering=true --xla_gpu_enable_triton_softmax_fusion=false \
+ --xla_gpu_enable_all_gather_combine_by_dim=false --xla_gpu_enable_reduce_scatter_combine_by_dim=false \
+ --xla_disable_hlo_passes=rematerialization
+EOF
+
+
+# 1 node, DATA_DP=1, ICI_FSDP=8
+python3 xpk/xpk.py workload create --cluster ${CLUSTER_NAME} --workload ${WORKLOAD_NAME} \
```
Comment (rwitten) [inline MaxText/configs/a3/llama_2_7b/1vm.sh:36]: We don't put the XPK command inside of the script -- please match the style we use elsewhere.

Comment (rwitten) [reply MaxText/configs/a3/llama_2_7b/1vm.sh:36]: Also unless I'm missing something we coudl break this into separately 5 env files and 1 script?

**Review by rwitten — CHANGES_REQUESTED** [review]

File: MaxText/configs/a3/llama_2_7b/16vm.sh
Comment (yangyuwei) [inline MaxText/configs/a3/llama_2_7b/16vm.sh:1]: Shall we update the `--xla_dump_to=gs://runner-maxtext-logs/yooh/llama2-70b-$(date +%Y-%m-%d-%H-%M)/HLO_dumps/` to get rid of personal directory and change `70b` to `7b`?

Comment (yangyuwei) [reply MaxText/configs/a3/llama_2_7b/16vm.sh:1]: The same for `MaxText/configs/a3/llama_2_7b/8vm.sh`.

File: MaxText/configs/a3/llama_2_7b/1vm.sh (hunk 1 above)
Comment (michelle-yooh) [reply MaxText/configs/a3/llama_2_7b/1vm.sh]: I believe we need 5 different env and configs.

File: MaxText/configs/a3/llama_2_7b/16vm.sh
Comment (michelle-yooh) [reply MaxText/configs/a3/llama_2_7b/16vm.sh:1]: Updated!

**Review by yangyuwei — APPROVED** [review]
Thanks Michelle for the changes!

**Review by rwitten — APPROVED** [review]

**Pull request merged**